root/etc/cont-init.d/98-crowdsec

```diff
@@ -44,9 +44,10 @@ fi
 sed -i -r "s|API_KEY=.*$|API_KEY=${CROWDSEC_API_KEY}|" "${CONFIG_PATH}crowdsec-nginx-bouncer.conf"
 sed -i -r "s|API_URL=.*$|API_URL=${CROWDSEC_LAPI_URL}|" "${CONFIG_PATH}crowdsec-nginx-bouncer.conf"
 
+# Sed in CROWDSEC_MODE mode, if defined in the env, defaults to live
+sed -i -r "s|MODE=.*$|SECRET_KEY=${CROWDSEC_MODE-live}|" "${CONFIG_PATH}crowdsec-nginx-bouncer.conf"
```

Comment:
This will replace `MODE=` with `SECRET_KEY=`.

`${CROWDSEC_MODE-live}` should really be `${CROWDSEC_MODE:-live}` otherwise it will allow a null value.